components/src/AppHeader/appheader.main.tsx

```diff
@@ -289,6 +313,18 @@ class AppHeaderMain extends Component<AppHeaderMainProps, AppHeaderMainState> {
             </Link>
           </div>
 
+          <div className="links-container">
+            <RequisitionListsLink />
```

Comment:
This shows up even if a user is not logged in yet. We'll need to hide it until login. We can discuss with @BonnieEP on the best approach/location of this as we're planning to update the navigation as well for b2b.